### PR TITLE
thewhaleking/further-legacy-torch-improvements

### PR DESCRIPTION
**Pull request by thewhaleking**
Further implements the improvements of #1917

### CODE REVIEW

File: bittensor/commands/root.py [hunk 1]
```diff
@@ -405,7 +400,7 @@ def run(cli: "bittensor.cli"):
     @staticmethod
     def _run(cli: "bittensor.cli", subtensor: "bittensor.subtensor"):
         wallet = bittensor.wallet(config=cli.config)
-        subnets: List[bittensor.SubnetInfo] = subtensor.get_all_subnets_info()
+        # subnets: List[bittensor.SubnetInfo] = subtensor.get_all_subnets_info()
```
Comment (opendansor) [inline bittensor/commands/root.py]: Why comment this out instead of removing it?

File: bittensor/commands/root.py [hunk 2]
```diff
@@ -283,7 +282,7 @@ def run(cli: "bittensor.cli"):
     def _run(cli: "bittensor.cli", subtensor: "bittensor.subtensor"):
         r"""Set weights for root network."""
         wallet = bittensor.wallet(config=cli.config)
-        subnets: List[bittensor.SubnetInfo] = subtensor.get_all_subnets_info()
+        # subnets: List[bittensor.SubnetInfo] = subtensor.get_all_subnets_info()
```
Comment (basfroman) [inline bittensor/commands/root.py]: let's delete this. I checked, it just makes unnecessary code and doesn't have any effect.
the same below

**Review by basfroman — APPROVED** [review]
not blocking comments
LGTM

File: bittensor/commands/root.py (hunk 1 above)
Comment (thewhaleking) [reply bittensor/commands/root.py]: Good call. Forgot to fully remove them after testing.

**Pull request merged**